test/gtest/uct/uct_test.h

```diff
@@ -278,7 +278,6 @@ std::ostream& operator<<(std::ostream& os, const resource* resource);
     tcp,                     \
     mm,                      \
     cma,                     \
-    cuda_ipc,                 \
```

Comment:
why is it removed?

Reply:
cuda IPC is not expected to work on same process context.  
Earlier, it is checking if  remote context and local cuda context are same or not and then skipping the cuda IPC flow and during cudacopy directly.  So, in gtest( single process) scenario, we were never really testing cuda-IPC.
And also, In this PR we removed cuda-ipc same ctx check as that is not necessary.

I think for now, will relay on jenkins/perftest for testing.